src/backend/generator.js

```diff
@@ -241,11 +246,20 @@ exports.createDistDir = function(req, socket, callback) {
       console.log('================================WRITING\n');
       if (emit) socket.emit('live.process', {donePercent: 70, status: "Compiling the HTML pages from templates" });
 
-      getJsonData(req.body, function(data) {
+      getJsonData(req.body, function(error, data) {
+        if (error) {
+          console.log('Error Invalid Zip');
```

Comment:
@poush Thanks for the review. The error message has already been added to the error logs.